spec/lib/cyclonedx/cyclonedx_report_spec.rb

```diff
@@ -31,6 +31,15 @@
       expect(Cyclonedx::Report.validate_cyclonedx(report)).to eq(report)
     end
 
+    it 'succeeds if scan_reports contain duplicate reports' do
+      scanner = Salus::Scanners::ReportRubyGems.new(repository: repo, config: {})
+      scanner.run
+
+      cyclonedx_reports = Cyclonedx::Report.new([[scanner.report, false], [scanner.report, false]],
```

Comment:
Does this rubygem report have duplicates?

Reply:
If so, a small comment to that end would be nice 👍

Reply:
If not, engineering/coinbase has duplicates and we can make a fixture from that

Reply:
Thank you for the suggestion. rubygem report itself does not have any duplicates. What we did here is to provide same report 2 times to `Cyclonedx::Report.new`. This simulates duplication.

@joshua-ostrom:
Do you want to me create a new fixture with duplicate or a simple comment describing that we are initializing `Cyclonedx::Report.new` with 2 same scanner.report should be fine for now? 🤔

Reply:
Simple comment works for me!  Thank you

Reply:
Updated. Can you please review again? Thank you.